domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/LayoutServiceImpl.java

```diff
@@ -59,4 +54,36 @@ public void createDisplayGroups(List<DisplayGroupEntity> displayGroups) {
         }
         displayGroupRepository.saveAll(displayGroups);
     }
+
+    private void validate(List<GenericLayoutEntity> genericLayouts) {
+        ValidationResult result = new ValidationResult();
+        for (GenericLayoutEntity genericLayoutEntity : genericLayouts) {
+            for (GenericLayoutValidator validator : genericLayoutValidators) {
+                result.merge(validator.validate(genericLayoutEntity));
+            }
+        }
+
+        for (GenericLayoutEntity genericLayoutEntity : genericLayouts) {
```

Comment:
can this validation for uniqueness be moved into a new GenericLayoutValidator?

Reply:
Moved all validation to GenericLayoutEntityValidatorImpl